Realm/Realm/Handles/SessionHandle.cs

```diff
@@ -256,5 +290,17 @@ private static void HandleSessionWaitCallback(IntPtr taskCompletionSource, int e
                 tcs.TrySetException(new RealmException(OuterMessage, inner));
             }
         }
+
+        [MonoPInvokeCallback(typeof(NativeMethods.SessionPropertyChangedCallback))]
+        private static void HandleSessionPropertyChangedCallback(IntPtr propertyChangedCallbackHandle, NotifiableProperties propertyChanged)
+        {
+            ((Action<string>)GCHandle.FromIntPtr(propertyChangedCallbackHandle).Target)(propertyChanged.ToString());
+        }
+
+        // !! The names of this enum must be kept in sync with the notifiable property names of Session !!
+        private enum NotifiableProperties : byte
```

Comment:
```suggestion
        private enum NotifiableProperty : byte
```

By convention, c# enums are named plural only if they are flag enums.